tools/launcher/common/eagle3/train_eagle_streaming.sh

```
@@ -0,0 +1,187 @@
+#   SERVE_PORT          default 8765
+#   SERVE_GPU           CUDA_VISIBLE_DEVICES for vllm serve. default "0"
+#   SERVE_TP            tensor-parallel-size for vllm serve. default 1
+#   SERVE_GPU_MEM_UTIL  default 0.4
+#   TRAIN_GPUS          CUDA_VISIBLE_DEVICES for the trainer. default = all
+#                       local GPUs except SERVE_GPU.
+#
+# All script args after the env-var setup are forwarded to launch_train.sh
+# (typically: --config <yaml> plus OmegaConf dotlist overrides).
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source ${SCRIPT_DIR}/../service_utils.sh
```

Shellcheck (0.11.0)

[info] Not following: ./../service_utils.sh was not specified as input (see shellcheck -x).

(SC1091)